docs/concepts.md

```diff
@@ -2,20 +2,30 @@
 
 Fleet is fundamentally a set of Kubernetes custom resource definitions (CRDs) and controllers
 to manage GitOps for a single Kubernetes cluster or a large scale deployment of Kubernetes clusters
-(up to one million). Below are some of the concepts of Fleet that will be useful through out this documentation.
+(up to one million). 
```

Comment:
We should likely change this to "tested up to one million clusters", but better wording than what I just wrote 😅. I think we should make it clear that Fleet's design is _technically_ mass scalable (horizontal), but we have _tested_ that amount.

Reply:
Revised per feedback.